Resources/enumerations/market-indexoption.html

```diff
@@ -1,3 +1,3 @@
-<p>The following <code>Market</code> enumeration members are available for Index Options:</p>
+<p>The following <code>Market</code> enumeration members are available for Indices Options:</p>
```

Comment:
"Indices" is good in most cases, but in the case of "Indices Options", it should really be "Index Options"